vmdb/app/models/ems_refresh/parsers/foreman.rb

```diff
@@ -92,6 +92,8 @@ def configured_system_inv_to_hashes(recs, profiles, operatingsystems)
             :hostname                   => cs["name"],
             :configuration_profile      => id_lookup(profiles, cs, "hostgroup_id"),
             :operating_system_flavor_id => id_lookup(operatingsystems, cs, "operatingsystem_id"),
+            :last_checkin               => cs["last_compile"],
+            :build_state                => cs["build"] == "true" ? "pending" : "finished",
```

Comment:
I think "pending" or nil would be better.  "finished" implies that some action has taken place.